ocaml/xapi/xapi_udhcpd.ml

```diff
@@ -162,7 +162,18 @@ let restart_nolock () =
   let pid = try Unixext.pidfile_read !Xapi_globs.udhcpd_pidfile with _ -> None in
   Opt.iter Unixext.kill_and_wait pid;
   let (_: string * string) = execute_command_get_output !Xapi_globs.busybox [ "udhcpd"; !Xapi_globs.udhcpd_conf ] in
```

Comment:
I wonder whether this should be a systemd service, that way we'd know we do not have multiple copies running, but this PR fixes the immediate problem so we can do that later.

Reply:
I'd worry then about whether the bridge exists or not - perhaps @robhoes could comment?

Reply:
So you mean a systemd service that is not included in the host boot sequence, but still (re)started from this function? I think that would work.

Reply:
Ah I see, yes that would make sense